src/views/home/components/header/header.styles.ts

```diff
@@ -10,17 +10,26 @@ const useHeaderStyles = createUseStyles((theme: Theme) => ({
     alignItems: "center",
     margin: [theme.spacing(2), "auto", 0],
   },
-  logoWrapper: {
+  leftBlock: {
     flex: 1,
-    height: 56,
     display: "flex",
-    alignItems: "center",
-    textAlign: "center",
-    flexDirection: "column",
+    justifyContent: "left",
+    gap: theme.spacing(0.75),
   },
-  appName: {
-    marginTop: theme.spacing(0.5),
-    color: theme.palette.black,
+  centerBlock: {
```

Comment:
If the other blocks have specific `flex: 1`, you can leave this one just with `display: flex` or remove it and assume and small pixel in the bottom

Reply:
Yep, I consciously added flex 1 to the 3 blocks to be explicit about them having exactly the same size.